src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -85,7 +86,7 @@ public CommandResult execute(FoodList foodList) throws DukeException {
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
```

Comment:
if using another check, this part also needs to be deleted

Reply:
remember to remove the above unused package as well

Reply:
Good notice!